### ISSUE TITLE
E471: Argument required on using :so

### ISSUE THREAD

**Issue by 0xd1r** [ThePrimeagen/init.lua#36]
## Summary

Hi, I'm a vim novice, so I'm not sure why this error comes up and I haven't found a comprehensive solution on the internet so far. Upon creating the plugins `remap.lua`, using `:so` to source the file returns `E471: Argument required`.

![image](https://user-images.githubusercontent.com/92980894/213341958-5180097c-275f-4b72-b0af-256098280b05.png)

<br>

## Issue
Command used: `:so`
Location: File `remap.lua`
<br>

Any fix would be greatly appreciated.
:)
<br>

**Comment by alitokur** [comment]
hey @0xd1r  
did you try to do a fresh install?

**Comment by 0xd1r** [comment]
Yeah, I just built 0.9.x from source. Having the issue. @alitokur

**Comment by alitokur** [comment]
before the installation maybe you can remove other configs. 

```
rm -rf nvim
rm -rf ~/.local/share/nvim
rm -rf ~/.local/state/nvim
```
I think the root cause is vim. cmd.Ex, but to be sure, can u remove the second line? just try :so with the first line. 

`vim.g.mapleader = " "`

**Comment by ThePrimeagen** [comment]
can you do `vim` and print out the version it displays?

(or just do :version in command mode)
